tests/Behat/Element/Admin/OrderCreateFormElement.php

```diff
@@ -270,4 +270,11 @@ private function waitForFormToLoad(): void
             return !$form->hasClass('loading');
         });
     }
+
+    public function isAddPaymentButtonVisible(): bool
+    {
+        return $this->getElement('payments')->find(
+            'css', '[data-form-collection="add"]') !== null
+        ;
```

Comment:
It still looks bad (I know, I'm annoying xD)

`return $this->getElement('payments')->find('css', '[data-form-collection="add"]') !== null;`

or

```
return
    $this->getElement('payments')->find('css', '[data-form-collection="add"]') !== null
;
```

It should fit in one line